configuration.go

```diff
@@ -149,11 +149,17 @@ func (ep *EntryPoints) Set(value string) error {
 		compress = strings.EqualFold(result["Compress"], "enable") || strings.EqualFold(result["Compress"], "on")
 	}
 
+	proxyprotocol := false
+	if len(result["ProxyProtocol"]) > 0 {
+		proxyprotocol = strings.EqualFold(result["ProxyProtocol"], "enable") || strings.EqualFold(result["ProxyProtocol"], "on")
```

Comment:
It may be good to create our own [`ParseBool`](https://golang.org/src/strconv/atob.go?s=351:391#L1) function that adds `on` and `enable`